python/cudf/cudf/tests/test_stats.py

```diff
@@ -372,3 +373,11 @@ def test_corr1d(data1, data2):
     got = gs1.corr(gs2)
     expected = ps1.corr(ps2)
     np.testing.assert_approx_equal(got, expected, significant=8)
+ 
+def test_df_corr():
+    gdf = cudf.datasets.randomdata(1000000, {str(x):float for x in range(50)})
```

Comment:
Could you also reduce the size? The idea is for these tests to be small to run fast and make debugging as easy as possible.
```suggestion
    gdf = randomdata(100, {str(x):float for x in range(50)})
```